packages/web/package.json

```
@@ -31,11 +31,11 @@
     "css:prefix": "postcss --config ./config/postcss.config.js --replace \"dist/css/*.css\" \"!dist/css/*.min.css\"",
     "css:minify": "cleancss --format breaksWith=lf --source-map-inline-sources --batch --batch-suffix \".min\" \"dist/css/*.css\" \"!dist/css/*.min.css\"",
     "js": "npm-run-all js:compile",
-    "js:compile": "npm-run-all --aggregate-output --parallel js:compile:**",
+    "js:compile": "npm-run-all --aggregate-output --parallel 'js:compile:**'",
     "js:compile:standalone": "rollup --environment BUNDLE:false --config config/rollup.config.js",
     "js:compile:standalone:esm": "rollup --environment ESM:true,BUNDLE:false --config config/rollup.config.js",
     "js:compile:bundle": "rollup --environment BUNDLE:true --config config/rollup.config.js",
-    "js:compile:plugins": "tsc --module es2015 --outDir dist/js --project ./config/tsconfig.prod.json",
+    "js:compile:plugins": "tsc --module es2015 --outDir dist/js --project ./config/tsconfig.prod.json && yarn postjs:compile:plugins",
     "postjs:compile:plugins": "cd dist/js && shx mv index.esm.js index.js && shx mv index.esm.js.map index.js.map && shx mv index.esm.d.ts index.d.ts && rm index.umd.js index.umd.js.map index.umd.d.ts",
```

Again, this is about copying:

```suggestion
    "js:compile:plugins": "tsc --module es2015 --outDir dist/js --project ./config/tsconfig.prod.json && yarn js:copy:plugins",
    "js:copy:plugins": "cd dist/js && shx mv index.esm.js index.js && shx mv index.esm.js.map index.js.map && shx mv index.esm.d.ts index.d.ts && rm index.umd.js index.umd.js.map index.umd.d.ts",
```

---

Fixed.